packages/mdc-menu-surface/foundation.js

```diff
@@ -425,8 +425,8 @@ class MDCMenuSurfaceFoundation extends MDCFoundation {
     }
 
     for (const prop in position) {
-      if (position.hasOwnProperty(prop) && position[prop] !== '0') {
-        position[prop] = `${parseInt(position[prop], 10)}px`;
+      if (position.hasOwnProperty(prop)) {
```

Comment:
This is changing behavior such that `position[prop]` is set even if `position[prop]` is 0. Do we want that?

Reply:
The existing code ensured that we didn't add a `"px"` suffix to values of `0`.

That's no longer necessary, so I removed this entire `for` loop.